ui/page/dexclient/markets.go

```diff
@@ -79,13 +101,50 @@ func (pg *Page) Layout(gtx C) D {
 		}
 	default:
 		body = func(gtx C) D {
-			return pg.pageSections(gtx, pg.registrationStatusLayout)
+			dex := pg.dex()
+			if !dex.Connected {
+				return D{}
+			}
+
+			if dex.PendingFee == nil && pg.miniTradeFormWdg != nil {
```

Comment:
Is it possible for `dex.PendingFee` to be nil and `pg.miniTradeFormWdg` is also nil? If not, `if dex.PendingFee == nil {` is enough condition.

I'd probably even go with `if dex.PendingFee != nil { show registrationStatusLayout }`. Then show `miniTradeFormWdg` lastly.